code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -298,7 +298,6 @@
 				true_offerings.Add(I)
 
 		if(num_item < req_num)
-			var/obj/item = path
```

Comment:
this variable was being declared but is not used at all.

Reply:
Oh, I have solved the mystery now.